pkg/runtime/runtime.go

```diff
@@ -1845,10 +1908,45 @@ func (a *DaprRuntime) processComponents() {
 				log.Fatalf(e)
 			}
 			log.Errorf(e)
+		} else if exists {
+			log.Infof("process component %s is replaced with a newer version, the older version will be shutdown", comp.Name)
```

Comment:
As we agreed, let's just not do the update of the component in this fix. Reloading components dynamically requires further planning and testing.